packages/core/src/payment/payment-strategy-action-creator.spec.ts

```diff
@@ -64,7 +67,7 @@ describe('PaymentStrategyActionCreator', () => {
     let state: CheckoutStoreState;
     let store: CheckoutStore;
     let strategy: PaymentStrategy;
-    let noPaymentDataStrategy: PaymentStrategy;
+    let noPaymentDataStrategy: NoPaymentDataRequiredPaymentStrategy;
```

Comment:
Does it have to be a type of `NoPaymentDataRequiredPaymentStrategy`, or can it simply be of type `PaymentStrategyV2` ?

Reply:
Yes, we can. So I have removed `NoPaymentDataRequiredPaymentStrategy` from `no-payment-integration` export.